parsec/crypto.py

```diff
@@ -157,27 +157,47 @@ def encrypt_for_self(self, data: bytes) -> bytes:
 # Helpers
 
 
+# Binary encoder/decoder for url use.
+# Notes:
+# - We replace padding char `=` by a simple `s` (which is not part of
+#   the base32 table so no risk of collision) to avoid copy/paste errors
+#   and silly escaping issues when carrying the key around.
+# - We could be using base64url (see RFC 4648) which would be more efficient,
+#   but backward compatibility prevent us from doing it :'(
```

Comment:
Funny how `base64.urlsafe_b64encode` still includes some `=`: 
```python
In [3]: base64.urlsafe_b64encode(b"a")
Out[3]: b'YQ=='
```

Reply:
yes, ulrsafe is not about the `=` character but about removing `+` and `/` (see https://docs.python.org/3/library/base64.html#base64.urlsafe_b64decode)

copy/paste error is much less of an issue now that the invite link is passed trough mail